components/views/auth/ResetPassword.tsx

```diff
@@ -60,7 +136,7 @@ const ResetPassword = () => {
                         </FormRow>
                         <SubmitButton
                             label="Submit"
-                            disabled={!isFormValid}
+                            disabled={disableSubmit}
```

Comment:
`disabled={user.loading || disableSubmit}`

Reply:
Honestly, now that i think about it. I'm not sure if i did that in the other forms, would you mind double checking and adding that if i missed it?